plugins/win-wasapi/win-wasapi.cpp

```diff
@@ -252,13 +252,13 @@ HRESULT WASAPISource::InitDevice(IMMDeviceEnumerator *enumerator)
 		return res;
 
 	if (!device_name.empty()) {
-		blog(LOG_INFO, "[WASAPISource::InitDevice]: Failed to init device and device name not empty",
-		     device_name.c_str());
+		blog(LOG_INFO, "[WASAPISource::InitDevice][%08X]: Failed to init device and device name not empty",
+		     this, device_name.c_str());
```

Comment:
Should we remove `device_name.c_str()`? It's not being logged no?

Reply:
fixed. made it to print device name.